qa/vagrant/src/test/resources/packaging/scripts/80_upgrade.bats

```diff
@@ -37,6 +37,11 @@ load os_package
 # Cleans everything for the 1st execution
 setup() {
     skip_not_dpkg_or_rpm
+
+    sameVersion=
```

Comment:
Can you make this `sameVersion=""` or `sameVersion="false"` or something like that?